### DISCUSSION TITLE
magento.test not loading on windows 11

### DISCUSSION THREAD

**Discussion by climba** [markshust/docker-magento#1239] [Q&A]
Ive been following the tutorial over at [https://courses.m.academy/courses/487758/lectures/8974570](https://courses.m.academy/courses/487758/lectures/8974570)

Everything runs error free but when I navigate to magento.text I get the blank failed to load this page error as though nothings there. 

Are there any extra steps needed for windows in terms of editing the host file?

![Screenshot 2024-10-31 195237](https://github.com/user-attachments/assets/1aa5f587-7885-435d-82ea-4467a74a1992)

**Comment by its-leofisher** [comment]
What does your host file look like?  and are you running under WSL?  I had a lot of issues running with Windows I ended up using WSL, this guide can help you: https://github.com/markshust/docker-magento/discussions/372
